src/opnsense/mvc/app/models/OPNsense/Syslog/Syslog.xml

```diff
@@ -0,0 +1,146 @@
+            <MinimumValue>5</MinimumValue>
+            <ValidationMessage>Log file size must be a positive integer greater than 5.</ValidationMessage>
+        </RotationFileSize>
+
+        <RotationCount type="IntegerField">
+            <default>5</default>
+            <Required>Y</Required>
+        </RotationCount>
+
+        <LogWebServer type="BooleanField">
+            <default>0</default>
+            <Required>Y</Required>
+        </LogWebServer>
+
+        <Firewall>
+            <LogDefaultBlock type="BooleanField">
+                <default>0</default>
+                <Required>Y</Required>
+            </LogDefaultBlock>
+            <LogDefaultPass type="BooleanField">
+                <default>0</default>
+                <Required>Y</Required>
+            </LogDefaultPass>
+            <LogBogons type="BooleanField">
+                <default>0</default>
+                <Required>Y</Required>
+            </LogBogons>
+            <LogPrivateNets type="BooleanField">
+                <default>0</default>
+                <Required>Y</Required>
+            </LogPrivateNets>
+            <FilterDescriptions type="OptionField">
+                <default>val_0</default>
+                <Required>Y</Required>
+                <OptionValues>
+                    <val_0>Omit descriptions</val_0>
+                    <val_1>Display as column</val_1>
+                    <val_2>Display as second row</val_2>
+                </OptionValues>
+            </FilterDescriptions>
+       </Firewall>
+
+        <LocalSockets>
+            <Socket type="ArrayField">
+                <Path type="TextField">
+                    <Required>Y</Required>
+                </Path>
+            </Socket>
+        </LocalSockets>
+
+        <LogCategories>
+            <Category type="ArrayField">
+                <Name type="TextField">
+                    <Required>Y</Required>
+                </Name>
+                <Description type="TextField">
+                    <Required>Y</Required>
+                </Description>
+                <LogRemote type="BooleanField">
+                </LogRemote>
+            </Category>
+        </LogCategories>
+
+        <Remote>
+            <Enable type="BooleanField">
+                <default>0</default>
+                <Required>Y</Required>
+            </Enable>
+            <SourceIP type="InterfaceField">
+                <Required>N</Required>
+                <default></default>
+            </SourceIP>
+            <Proto type="OptionField">
+                <Required>N</Required>
+                <default>ipv4</default>
+                <OptionValues>
+                    <ipv4>IPv4</ipv4>
+                    <ipv6>IPv6</ipv6>
+                </OptionValues>
+            </Proto>
+            <BindAddress type="TextField">
+            </BindAddress>
+            <Servers type="CSVListField">
+                <Required>N</Required>
+                <mask>/^([a-zA-Z0-9.:,]){0,}/</mask>
```

Comment:
* "i" to make it not case sesitive
* {0,} is the same as *
* the Dot may need to be escaped - otherwise it would not make sense to have the other characters inside
* Is the bracket required for validation?